docs/program/run.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Run SDK Code"
+linkTitle: "Run Code"
+weight: 50
+type: "docs"
+description: "Execute the logic you've written to control your robot or fleet."
+images: ["/services/img/icons/sdk.svg"]
+tags: ["client", "sdk", "application"]
+---
+
+After saving your boilerplate code sample and adding control logic with Viam's API methods, run your program to control your Viam-connected robot.
+Do this by running a command to execute the program in the terminal of a machine with the appropriate programming language and Viam SDK installed.
+
+For example:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+python3 ~/myCode/myViamFile.py
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+go run ~/myCode/myViamFile.py
+```
+
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+For an example, see [this execution demo.](https://github.com/viamrobotics/viam-typescript-sdk/tree/main/examples/vanilla)
+
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+For information on running C++ code see [the instructions on GitHub](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+If you are connecting to your robot through an external signaling address, as is the case in the boilerplate code samples generated by the Viam app, you can perform this step on any computer.
+As long as this computer is able to establish a network connection with the robot's computer, the control logic will be executed on the robot.
+
+### Run Code Locally
+
+If you need to run [PID control loops](https://en.wikipedia.org/wiki/PID_controller) or other on-robot code, you can run control code on the same board that is running `viam-server`.
+
+To ensure intermittent internet connectivity does not interfere with the code's execution, there are some special steps you need to follow:
```

Comment:
OK, so honest question - I often run code locally on my robot just so I don't need my laptop to always be up and running etc - but in many cases the robot is in a place of good network connectivity so I don't take these special steps.  Should we be telling people to do this whenever they are running the code on the robot, or just if they think their robot will sometimes lose connectivity etc?

Reply:
@npentrel Might have a better idea than I do but I think "ensure" is the key word, so we could add a note explaining that this is only necessary if you think your robot will sometimes lose connectivity?